### PR TITLE
Add torchaudio

### PR DESCRIPTION
**Pull request by kobrineli**

### CODE REVIEW

File: projects/torchaudio/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    git checkout 0f1621df1a0a73956c7ce4e2f72f069e610e0137 && \
+    git submodule update --init --recursive --jobs 0
+
+COPY torchaudio.patch /
+RUN git clone https://github.com/pytorch/audio.git audio
+RUN cd /audio && \
+    git checkout 30afaa9b2f460ca3baf6ac177bce4f6b5da8b99c && \
+    git apply /torchaudio.patch
+
+COPY build.sh torchaudio.patch /
```
Comment (apach301) [inline projects/torchaudio/Dockerfile:33]: `torchaudio.patch` have already been copied

Comment (kobrineli) [reply projects/torchaudio/Dockerfile]: fixed

**Review by apach301 — APPROVED** [review]

File: projects/torchaudio/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Torchaudio
+
+This library is part of the PyTorch project. PyTorch is an open source machine learning framework.
+The aim of torchaudio is to apply PyTorch to the audio domain.
+
+## Build Docker
+
+    $ sudo docker build -t oss-sydr-fuzz-torchaudio.
```
Comment (PaDarochek) [inline projects/torchaudio/README.md:8]: Add space between `oss-sydr-fuzz-torchaudio` and `.`

File: projects/torchaudio/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+RUN git clone https://github.com/pytorch/audio.git audio
+RUN cd /audio && \
+    git checkout 30afaa9b2f460ca3baf6ac177bce4f6b5da8b99c && \
+    git apply /torchaudio.patch
+
+COPY build.sh /
+COPY load_audio.cc /audio
+
+RUN CONFIGS="libfuzzer afl" /build.sh
+RUN CONFIGS="sydr coverage" /build.sh
```
Comment (PaDarochek) [inline projects/torchaudio/Dockerfile:37]: Maybe we can join these two lines and invoke running build.sh once?

**Review by PaDarochek — CHANGES_REQUESTED** [review]

File: projects/torchaudio/Dockerfile (hunk 3 above)
Comment (kobrineli) [reply projects/torchaudio/Dockerfile:37]: These steps were separated on purpose to save steps results in 2 different intermediate containers.

File: projects/torchaudio/README.md (hunk 2 above)
Comment (kobrineli) [reply projects/torchaudio/README.md:8]: fixed

File: projects/torchaudio/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+    # cd /fuzz
+
+Run hybrid fuzzing:
+
+    # sydr-fuzz -c load_audio.toml run
+
+Minimize corpus:
+
+    # sydr-fuzz -c load_audio.toml run
```
[the remark this answers is not in the document]
Comment (kobrineli) [reply projects/torchaudio/README.md]: fixed

**Pull request merged**